bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/i18n/sncf.properties

```diff
@@ -0,0 +1,29 @@
+
+binding.sncf.name = SNCF Binding
+binding.sncf.description = Retrieves French railway informations
+
+config.thing-type.sncf.api.apiID.label = API ID
+config.thing-type.sncf.api.apiID.description = Your Sncf API ID
+
+thing-type.sncf.api.label = SNCF API
+thing-type.sncf.api.description = This bridge represents the gateway to Sncf API.
+
+thing-type.sncf.station.label = Station
+thing-type.sncf.station.description = This represents a station hosting some transportation mode.
+  
+channel-type.sncf.direction.label = Direction
+channel-type.sncf.direction.description = The direction of this route.
+channel-type.sncf.code.label = Code
+channel-type.sncf.code.description = Code of the line.
+channel-type.sncf.name.label = Name
+channel-type.sncf.name.description = Name of the line.
+channel-type.sncf.network.label = Network
+channel-type.sncf.network.description = Name of the transportation network.
+channel-type.sncf.commercialMode.label = Commercial Mode
+channel-type.sncf.commercialMode.description = Commercial modes are close from physical modes, but not normalized and can refer to a brand, something that can be specific to a network, and known to the traveler.
+channel-type.sncf.timestamp.label = Timestamp
+channel-type.sncf.timestamp.description = Timestamp of the future event.
+
```

Comment:
Group labels are missing.

The best is that you use @wborn tool to generate this file.

Reply:
Yes it's a handy tool! :slightly_smiling_face: